src/parser/jobs/mnk/modules/RiddleOfFire.tsx

```diff
@@ -1,198 +1,208 @@
 import {t} from '@lingui/macro'
 import {Plural, Trans} from '@lingui/react'
-import {DataLink} from 'components/ui/DbLink'
-import {RotationTable} from 'components/ui/RotationTable'
+import {ActionLink} from 'components/ui/DbLink'
 import {Action} from 'data/ACTIONS'
-import {Event, Events} from 'event'
-import {Analyser} from 'parser/core/Analyser'
-import {EventHook} from 'parser/core/Dispatcher'
-import {filter} from 'parser/core/filter'
 import {dependency} from 'parser/core/Injectable'
-import {Data} from 'parser/core/modules/Data'
-import Suggestions, {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
-import {Timeline} from 'parser/core/modules/Timeline'
+import {
+	BuffWindow,
+	EvaluatedAction,
+	EvaluationOutput,
+	ExpectedGcdCountEvaluator,
+	LimitedActionsEvaluator,
+	WindowEvaluator,
+} from 'parser/core/modules/ActionWindow'
+import {HistoryEntry} from 'parser/core/modules/ActionWindow/History'
+import {GlobalCooldown} from 'parser/core/modules/GlobalCooldown'
+import {SEVERITY, Suggestion, TieredSuggestion} from 'parser/core/modules/Suggestions'
 import React from 'react'
 import {BLITZ_SKILLS} from './constants'
 import {DISPLAY_ORDER} from './DISPLAY_ORDER'
-import {fillActions} from './utilities'
 
-// Expected GCDs and buffs
-// technically can get 2 tacles and should as much as possible, but don't ding for it in case it's for mechanics
+const TOTAL_GCD_AMOUNT = {
+	10: SEVERITY.MINOR,
+	9: SEVERITY.MEDIUM,
+	6: SEVERITY.MAJOR,
+}
+
+const MISSED_BROTHERHOOD_SEVERITY = {
+	1: SEVERITY.MEDIUM,
+	2: SEVERITY.MAJOR,
+}
+
+const MEDITATION_DURING_ROF_SEVERITY = {
+	1: SEVERITY.MEDIUM,
+	3: SEVERITY.MAJOR,
+}
+
 const EXPECTED = {
 	GCDS: 11,
-	BLITZES: 1,
```

Comment:
The only EXPECTED here is now GCDs, could prob just make this `const EXPECTED_GCDS = 11` and be done with it, although I'm not sure why this is needed since the expected GCD evaluator has the GlobalCooldown object yet also wants an expected field.